synapse_data_warehouse/synapse/dynamic_tables/V2.26.2__node_latest.sql

```diff
@@ -0,0 +1,36 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+CREATE DYNAMIC TABLE IF NOT EXISTS NODE_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+        WITH latest_unique_rows AS (
+            SELECT
+                *
+            FROM
+                {{database_name}}.synapse_raw.nodesnapshots --noqa: TMP
+            WHERE
+                SNAPSHOT_TIMESTAMP >= CURRENT_TIMESTAMP - INTERVAL '14 DAYS'
```

Comment:
This is the major difference between the `UPSERT_TO_NODE_LATEST_TASK` (log onto SYSADMIN to view this task on Snowsight) and my query. I found that not adding the 14 day window restriction slows down the query by _a lot_, so even though we can remove deleted records using `change_type`, leaving this `SNAPSHOT_TIMESTAMP` window will help with efficiency.